x-pack/plugins/cases/public/components/all_cases/all_cases_list.test.tsx

```diff
@@ -705,7 +739,7 @@ describe.skip('AllCasesListGeneric', () => {
     });
   });
 
-  describe('Actions', () => {
+  describe.each(Array(55).fill(null))('Actions', () => {
```

Comment:
don't forget to remove it before merge 😄

Reply:
Good point! Thanks!